### DISCUSSION TITLE
Grouping in table

### DISCUSSION THREAD

**Discussion by cdecinkoKnight** [quasarframework/quasar#16217] [General - Components / Directives / etc]
I saw discussions of a table grouping feature during Quasar v1 days but no mention for v2. Has anyone come up with a workable solution for grouping results in a single table, like sub categories or such?  Below the column header row, show a full wide row with "Company A" and then result rows below. Then a full wide row for "Company B" followed by those results.

**Comment by ivan-pinto** [comment]
#5981 and #1512
